spec/models/comment_spec.rb

```diff
@@ -21,4 +21,9 @@
     @comment.commenter = nil
     expect !@comment.valid?
   end
+	
```

Comment:
you could test for validations aswell: https://github.com/asm-products/mana-rails/blob/develop/spec/models/client_spec.rb#L6
https://github.com/thoughtbot/shoulda-matchers